vision/classification/mobilenet/README.md

```diff
@@ -16,6 +16,13 @@ The below model is using multiplier value as 1.0.
  |Model        |Download  |Download (with sample test data)| ONNX version |Opset version|Top-1 accuracy (%)|Top-5 accuracy (%)|
 |-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
 |MobileNet v2-1.0|    [13.6 MB](model/mobilenetv2-7.onnx)  |  [14.1 MB](model/mobilenetv2-7.tar.gz) |  1.2.1  | 7| 70.94    |     89.99           |
+|MobileNet v2-1.0-fp32|    [13.3 MB](model/mobilenetv2-12.onnx)  |  [12.9 MB](model/mobilenetv2-12.tar.gz) |  1.9.0  | 12| 69.48    |     89.26           |
+|MobileNet v2-1.0-int8|    [3.5 MB](model/mobilenetv2-12-int8.onnx)  |  [3.7 MB](model/mobilenetv2-12-int8.tar.gz) |  1.9.0  | 12| 68.30    |     88.44           |
```

Comment:
I have a question regarding to the ONNX version (1.9.0) here: by default, ONNX 1.9.0 produces ONNX model with opset_version=14 ([version map](https://github.com/onnx/onnx/blob/main/docs/Versioning.md#released-versions)). Is it more proper to use ONNX 1.7.0 here or you have other concern?

Reply:
The fp32 model I submitted is converted by the existing model, during conversion I ignored the version map and our tool suggests opset>11 so I directly used 12.
ONNX 1.9.0 is my validation env where I do conversion and quantization successfully. If it must follow the version map I can verify and then submit again.
Thanks.

Reply:
Thank you for the clarification. I saw some models did not follow version map either so I think it should be fine.